src/server/common/model/section/section-model/section-model.js

```diff
@@ -92,4 +96,26 @@ export class SectionModel {
 
     return new this(pages)
   }
+
+  // eslint-disable-next-line jsdoc/require-returns-check
+  /**
+   * @param {Request} req
+   * @returns {object}
+   */
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  buildGdsTaskDetails(req) {
```

Comment:
If we move the section template next to the section-model & give it the same name (`section-model.njk`), we could happily call this `viewModel` & it'd follow the pattern of the answer page.

just a thought!

Reply:
Do we have a section template? Maybe you are talking about the task list, but we also have the create summary macro so I'm not sure as both are related to sections.  Shall we leave this one as is and maybe have a think after?

Reply:
create summary macro is the equivalent of the session-model view (IMO).  In the same way that the input view is not the only place the answer data is used, but it is the main place. 

But for this PR, leave it as is and we can have a think after.